lib/lib.go

```diff
@@ -543,6 +543,13 @@ func NewInstanceFromConfigAndNode(cfg *config.Config, node *p2p.QriNode) *Instan
 	fsint := fsi.NewFSI(r, bus)
 	dc := dscache.NewDscache(ctx, r.Filesystem(), []hook.ChangeNotifier{r.Logbook(), fsint}, pro.Peername, "")
 
+	// TODO (b5) - lots of tests pass "DefaultConfigForTesting", which uses a different peername /
```

Comment:
the issue is the `DefaultConfig` and provided `repo` have different peernames, and this function doesn't generate an error. Because our codebase has multiple ways to get a peername, tests will get different answers depending on where they look. This became a bigger issue with the introduction of `lib.ResolveReference`, which uses `config.Profile.Peername` to replace the "me" keyword